upgrading/whats-new.md

```diff
@@ -4,19 +4,12 @@ description: This section discusses the new features introduced in the latest ve
 heading: What's New
 ---
 
-This section is for users who are using the previous version of FusionCharts in their application. Here, we'll talk about the new features introduced in v3.21.x.
+This section is for users using the previous version of FusionCharts in their application. Here, we'll talk about the new features introduced in v3.22.x.
 
 
-## Update for Ramda Support
+## Blazor Integration
 
-To improve our security, FusionCharts release 3.21 updated the `Ramda` dependency package from version 0.25.0 to version 0.29.0. 
-
-
-## Cross-Site Scripting
-
-FusionCharts version 3.21 implemented the new function to eliminate any JavaScript functions in your code that may cause an error. The new function removes any HTML-based predefined JavaScript functions (like `onError`, `onClick`, `onChange`, etc.) from the code and lets the charts and images display smoothly.  
-
-
-## New Maps
-
-FusionCharts version 3.21 introduces new maps for Argentina, Peru, and Costa Rica. With a total of 58 new maps, you can now drill down to each specific department within those countries. This version added all 27 Peruvian provinces, all 24 Argentinian provinces, and all seven Costa Rican provinces. 
+FusionCharts release version 3.22 introduces the Blazor integration with FusionCharts, which integrates with the FusionCharts Core JavaScript library using webview. Opening up new possibilities for dynamic and interactive charting within Blazor applications.
```

Comment:
webview? what is the source for this?

Reply:
This was taken from the first paragraph of the README file, but I just noticed that something is incorrect given that the README mentioned Flutter instead of Blazor. 

"This plugin provides Flutter interface which integrates with FC Core JavaScript library using webview. "

Reply:
The paragraph was reworded in the newest PR.